documentation/release-notes/topics/version_4.rst

```diff
@@ -2,6 +2,79 @@
+
+========================================
+Enhanced Feature: Alarm event management
+========================================
+The alarm grid has been improved to show the alarm event category.
+
+===================================================
+Enhanced Feature: Alarm event notification policies
+===================================================
+Alarm event categories have been added as a criteria for alarm events. A Hyperview administrator is now able to select a specific alarm event category or all categories when defining a policy.
+
+Alarm event category options are:
+
+- All Categories
+- AssetTracker Status
+- Data Collector Status
+- Firmware Status
+- Host Unreachable
+- Sensor Threshold Check
+- SNMP Trap
+- Unknown Trap
+
+Existing notification policies will have the "All Categories" option selected by default. This can be changed by editing the policy.
+
+=======================================================
+Enhanced Feature: Asset alarm event notification muting
+=======================================================
+Assets undergoing maintenance or causing excessive notifications can now be muted for a specific period of time. The functionality is available under the **Actions -> Mute Notifications** menu.
+
+A Mute action can be cancelled from the **Actions -> Cancel Mute Notifications** menu.
+
+================================================================
+Enhanced Feature: Bulk actions -> Mute/Cancel Mute Notifications
+================================================================
+Mute and Cancel Mute Notifications can also be performed as a bulk action on a list of assets. The bulk action is available from:
+
+- Advanced Search
+- Information -> Assets
+- Business Entity -> Associations -> Assets
+- Assets By Type
+
+===================================
+Enhanced Feature: Dashboard widgets
+===================================
+A widget was added to display asset documents and links.
+
+======================================
+Other notable changes and improvements
+======================================
+- The asset Actions menu has been enhanced to group related actions.
+- Server asset types can now be changed into Storage types and ViceVersa.
+- Various minor improvements to the Carbon Footprint report have been done.
+- Enhanced the Add New page of Assets, Connections and Circuits to detect unsaved changes and warn the user if they try to navigate to another page before saving.
+- Enhanced the Rack asset type to have Lifecycle information.
+- String Type custom properties will be made clickable if the user enters a valid URL.
+
+=================
+Notable bug fixes
+=================
+- **AS-14401** Fixed a bug that caused Circuit custom properties not appear in alphabetical order when added to a data grid.
```

Comment:
AS-14401 should be removed from this list since it is no longer scoped for this release

Reply:
Thanks, addressed.